src/ansible/roles/dns/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Install dnsmasq package
+  ansible.builtin.package:
```

Comment:
Please, avoid ansible.builtin prefix to follow coding style from other files.

Reply:
Ack